src/invoice2data/extract/invoice_template.py

```diff
@@ -174,11 +188,33 @@ def extract(self, optimized_str):
         output["issuer"] = self["issuer"]
 
         for k, v in self["fields"].items():
+            # k is the key of the field
+            # v is the value
             if isinstance(v, dict):
+                # Options were supplied to this field
+                if "area" in v and input_module == pdftotext:
```

Comment:
The Tesseract input module is using pdftotext now too. To format the output with the position of the original text.
https://github.com/invoice-x/invoice2data/blob/4d517d131fdaa568b4282f2f0a12abef98751075/src/invoice2data/input/tesseract.py#L100-L118

Does it make sense to add tesseract as an option, to also use the area support?